lib/dependency_spy/formatters/text.rb

```diff
@@ -48,6 +50,17 @@ def self.format(manifests)
         end
       end
 
+      def self.apply_style(formatted_output, severity_threshold)
```

Comment:
I would prefer to have this inside the Text formatter since it is a detail of the parser.
Would be also nice to disable colors with a flag and/or option.

Reply:
Done.